components/Summary/Summary.jsx

```diff
@@ -123,25 +128,15 @@ const Summary = ({
   formSteps,
   formPath,
   canEdit,
-  isSummaryCollapsable,
+  // isSummaryCollapsable,
```

Comment:
If the summary collapsable feature is going to get removed we should remove it also from the code

Reply:
it's not my intention to remove this feature, i've just commented these out for now so we can get the code that is here merged. it will be easy to re-add the expand/collapse all button later once the bulk of this is in